projects/core/src/model/consignment-tracking.model.ts

```diff
@@ -0,0 +1,20 @@
+export interface ConsignmentTrackingEvent {
+  eventDate?: Date;
```

Comment:
Could you confirm, that we really have a date here, when data is loaded from backend? I would guess it'll be a string of some kind.

Reply:
Yes, it's loaded from backend